lib/editors.coffee

```diff
@@ -55,7 +55,12 @@ class QuillJs extends Editor
   observeLocalText: (backend)->
     @editor.on "text-change", (deltas, source)->
       # call the backend with deltas
-      backend deltas.ops
+      position = backend deltas.ops
+      # trigger an extra event to move cursor to position of inserted text
+      @editor.selection.emitter.emit(
```

Comment:
I couldn't find any way to trigger an event, but I hope there's one (this is a hack).
Regarding your second point, it's already in editors.coffee…